types/aws-serverless-express/middleware.d.ts

```diff
@@ -1,24 +1,22 @@
-import { IncomingMessage } from 'http';
+import 'http';
 import { APIGatewayProxyEvent, Context } from 'aws-lambda';
 import { RequestHandler } from 'express';
 
-type NonFunctionPropertyNames<T> = { [K in keyof T]: T[K] extends Function ? never : K }[keyof T];  // tslint:disable-line:ban-types
-type NonFunctionProperties<T> = Pick<T, NonFunctionPropertyNames<T>>;
-
-declare module 'http' {
-  interface IncomingMessage {
-    apiGateway?: {
-      event: Omit<APIGatewayProxyEvent, 'body'>;
-      context: NonFunctionProperties<Context>;
-    } | undefined;
-  }
+declare module 'node:http' {
```

Comment:
@SimonSchick same question as in other pr, does it introduce Node v16 dependency?